avocado/core/nrunner/runnable.py

```diff
@@ -87,6 +87,9 @@ def __init__(self, kind, uri, *args, config=None, **kwargs):
         self.dependencies = kwargs.pop("dependencies", None)
         self.variant = kwargs.pop("variant", None)
         self.output_dir = kwargs.pop("output_dir", None)
+        #: list of (:class:`ReferenceResolutionAssetType`, str) tuples
+        #: expressing assets that the test will require in order to run.
+        self.assets = kwargs.pop("assets", None)
```

Comment:
If you are adding a new property to runnable, you should also update the `get_command_args` and `get_dict` methods to correspond to this new property.

Reply:
I'm a bit on the fence here.  While it would be easy to do so, I can not think how the assets information would be used in the runner implementation.  The fact is that, by the time a function such as `get_command_args()` is called, in order to generate a command line, a spawner would have already handled the test files and data files.  The runner itself, different than say, ``config`` or ``tags``, has no current use for that information.

If we add that info, we'll simply be construction longer command lines with no added benefits.